pkg/store/bucket.go

```diff
@@ -221,6 +224,18 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Help: "Compressed size of postings stored into cache.",
 	})
 
+	m.seriesLookupDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+		Name:    "thanos_bucket_store_index_series_lookup_duration_seconds",
+		Help:    "Time it takes to lookup series from a bucket to respond a query. It also includes the cache fetch and store operations.",
+		Buckets: []float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120},
+	})
+
+	m.postingsLookupDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+		Name:    "thanos_bucket_store_index_postings_lookup_duration_seconds",
```

Comment:
```suggestion
		Name:    "thanos_bucket_store_index_cachable_fetches_duration_seconds",
```

I would add `cachable` to put highlight that it includes cache (: